neuraxle/metaopt/auto_ml.py

```diff
@@ -319,7 +319,10 @@ def getmtimens(filename):
 
         for base_path in files:
             with open(base_path) as f:
-                trial_json = json.load(f)
+                try:
+                    trial_json = json.load(f)
+                except Exception as err:
+                    continue
```

Comment:
Why fail silently here? I would like, as a user of the library, to know the reason why it failed. Maybe print stack trace and also a good reason and an error. This code surprises me here.

Reply:
yeah I will print invalid json file

Reply:
basically, it fails if the json is not well constructed